data/sql/updates/db_world/2017_12_08_00.sql

```diff
@@ -5,7 +5,7 @@ CREATE PROCEDURE updateDb ()
 proc:BEGIN DECLARE OK VARCHAR(100) DEFAULT 'FALSE';
 SELECT COUNT(*) INTO @COLEXISTS
 FROM information_schema.COLUMNS
-WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'version_db_world' AND COLUMN_NAME = '2017_12_08_00';
+WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'version_db_world' AND COLUMN_NAME = '2017_11_21_00';
```

Comment:
why these changes?

Reply:
Because this sql was still causing problem with the updates since b527073fd84f335fa20ac3943618448f9a43e414. You forget to change that column_name :P